crypto/evp_extra/evp_extra_test.cc

```diff
@@ -2627,34 +2637,44 @@ TEST_P(PerKEMTest, RawKeyOperations) {
   ASSERT_FALSE(EVP_PKEY_kem_check_key(pkey_new.get()));
 }
 
+// Perform Known Answer Test (KAT) on known KEMs.
+// These tests access the deterministic EVP APIs for KeyGen and Encapsulation.
+// To perform KATs in KEMs we use a DRBG seeded with a given state "seed".
+// This seed is input into a DRBG, which outputs random "coins". These coins
+// have been generated within the KAT files, so that DRBG seeding and calling
+// can be performed outside of the library/KEM implementation.
+// The "coins" provided to key generation are "keypair_coins".
+// The "coins" provided to key encapsulation are "encap_coins".
 TEST_P(PerKEMTest, KAT) {
-
   std::string kat_filepath = "crypto/";
   kat_filepath += GetParam().kat_filename;
 
   FileTestGTest(kat_filepath.c_str(), [&](FileTest *t) {
     std::string count;
-    std::vector<uint8_t> seed, pk_expected, sk_expected, ct_expected, ss_expected;
+    std::vector<uint8_t> seed, keypair_coins, encap_coins, pk_expected,
+        sk_expected, ct_expected, ss_expected;
 
     ASSERT_TRUE(t->GetAttribute(&count, "count"));
     ASSERT_TRUE(t->GetBytes(&seed, "seed"));
```

Comment:
Does anything use this?

Reply:
removed in [3578aee](https://github.com/aws/aws-lc/pull/1578/commits/3578aeef15c7532dc44a9604f8d9dd2e7793dae3)